src/did/did.ts

```diff
@@ -53,7 +54,10 @@ class DIDDocument implements Did {
     let vm;
     switch (keyType) {
       case IKeyType.Ed25519VerificationKey2020: {
-        this.context = [constant['DID_' + keyType].DID_BASE_CONTEXT];
+        this.context = [
+          constant['DID_' + keyType].DID_BASE_CONTEXT,
+          constant['DID_' + keyType].DID_KEYAGREEMENT_CONTEXT,
```

Comment:
You should not take DID_KEYAGREEMENT_CONTEXT context here since you ar enot allowing keyagreement verrelationship